tensorflow/python/keras/activations.py

```diff
@@ -254,7 +252,7 @@ def deserialize(name, custom_objects=None):
 def get(identifier):
   if identifier is None:
     return linear
-  if isinstance(identifier, six.string_types):
+  if isinstance(identifier, ("".__class__, u"".__class__)):
```

Comment:
I'm not a fan of this; I feel like hiding this complexity is exactly what six.string_types is used for in the tf codebase.

Reply:
Yes it does seem complex but I tried reducing the number of imports.

Reply:
Everything in six is just a few lines of could so it could be easily inlined; I think its value lies instead in documenting at the call site that we're dealing with a py2 vs py3 difference.

Reply:
ok thanks for the review.